firmware/components/driver_display_ili9341/Kconfig

```diff
@@ -19,6 +19,11 @@ menu "Driver: ILI9341 LCD display"
 	config PIN_NUM_ILI9341_BACKLIGHT
 		depends on DRIVER_ILI9341_ENABLE
 		int "GPIO pin used for backlight"
+		
+	config DRIVER_ILI9341_SPI_SPEED
+		depends on DRIVER_ILI9341_ENABLE
+		int "SPI clock speed"
+		default 1000000

```

Comment:
Why is the new default 1MHz when previously it was set to 40MHz? Seems a bit drastic to me

Reply:
Some devices are less capable than others. While 40MHz should work I've seen some issues with crappy breadboard constructions. 1MHz is the failsafe default.